arch/configure.defaults

```diff
@@ -798,6 +798,30 @@ CPPFLAGS            = -D_UNDERSCORE -DBYTESWAP -DLINUX -DIO_NETCDF -DIO_BINARY -
 ARFLAGS             =
 CC_TOOLS            =
 
+########################################################################################################################
+#ARCH    Linux x86_64, Intel oneAPI compiler with icx   # serial serial_NO_GRIB2 dmpar dmpar_NO_GRIB2
+#
+COMPRESSION_LIBS    = CONFIGURE_COMP_L
+COMPRESSION_INC     = CONFIGURE_COMP_I
+FDEFS               = CONFIGURE_FDEFS
+SFC                 = ifx
+SCC                 = icx
+DM_FC               = mpif90 -f90=$(SFC)
+DM_CC               = mpicc -cc=$(SCC)
+FC                  = CONFIGURE_FC
+CC                  = CONFIGURE_CC
+LD                  = $(FC)
+FFLAGS              = -FR -convert big_endian
+F77FLAGS            = -FI -convert big_endian
+FCSUFFIX            = 
+FNGFLAGS            = $(FFLAGS)
+LDFLAGS             = 
+CFLAGS              = -w -Wno-implicit-int 
```

Comment:
We should remove these flags as they suppress errors that should be fixed. #245 addresses the syntax issues

Reply:
Okay

Reply:
I think i got it done now